src/components/messenger/conversation-actions/container.test.tsx

```diff
@@ -241,7 +231,7 @@ describe('ConversationActionsContainer', () => {
     describe('canViewDetails', () => {
       it('is false when one on one conversation', () => {
         const state = new StoreBuilder().withActiveConversation(
-          stubConversation({ isOneOnOne: true, isSocialChannel: false })
+          stubConversation({ totalMembers: 2, isSocialChannel: false })
```

Comment:
there's a case where `totalMembers` can be 2 but still the conversation still isn't oneOnOne. (mentioned in other comment)